streams/es_samsung.m3u

```diff
@@ -31,3 +31,5 @@ https://tastemade-es16tm-samsung.amagi.tv/playlist.m3u8
 https://the-pet-collective-international-es.samsung.wurl.tv/playlist.m3u8
 #EXTINF:-1 tvg-id="TraceSportStars.fr",Trace Sport Stars (1080p) [Geo-blocked]
 http://tracesportstars-samsunges.amagi.tv/hls/amagi_hls_data_samsunguk-tracesport-samsungspain/CDN/playlist.m3u8
+#EXTINF:-1 tvg-id="LosGreenenlaGranCiudadTV.es",Los Green en la Gran Ciudad TV (1080p)
+https://app.viloud.tv/hls/channel/18c256b17679f228651f26b303a23e76.m3u8
```

Comment:
wasn't able to watch even with Spanish VPN. it does respond tho

Reply:
Does not working.